mbed/CMakeLists.txt

```
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/mbed-os CACHE INTERNAL "")
+set(MBED_CONFIG_PATH ${CMAKE_CURRENT_BINARY_DIR} CACHE INTERNAL "")
+set(APP_TARGET mbed-bluetooth-power-consumption-benchmark)
+
+include(${MBED_PATH}/tools/cmake/app.cmake)
+
+add_subdirectory(${MBED_PATH})
+
+add_subdirectory(mbed-os-ble-utils)
+
+add_executable(${APP_TARGET})
+
+mbed_configure_app_target(${APP_TARGET})
+
+project(${APP_TARGET})
+
+target_include_directories(${APP_TARGET}
+    PRIVATE
+        ../shared/include
+)
+
+target_sources(${APP_TARGET}
+    PRIVATE
+        source/main.cpp
+        ../shared/source/test_base.cpp
```

That file doesn't exist. But, you are missing other sources:

```suggestion
        source/MbedBluetoothPlatform.cpp
        ../shared/source/BluetoothPlatform.cpp
        ../shared/source/PowerConsumptionTest.cpp
```